rpc/RPCSession.java

```diff
@@ -83,7 +85,11 @@ public boolean isOpen() {
     public void close() {
         if (isOpen.compareAndSet(true, false)) {
             pulse.cancel();
-            blockingGrpcStub.sessionClose(SessionProto.Session.Close.Req.newBuilder().setSessionId(sessionId).build());
+            try {
+                blockingGrpcStub.sessionClose(SessionProto.Session.Close.Req.newBuilder().setSessionId(sessionId).build());
+            } catch (StatusRuntimeException e) {
+                throw new GraknClientException(e);
+            }
```

Comment:
It was inconsistent that we threw `StatusRuntimeException` here, when the rest of our codebase throws `GraknClientException` on any server error.